### PR TITLE
Add French language

### PR DESCRIPTION
**Pull request by JUL1EN094**
Hi,

Try to add French language by doing same the than others previous language addition - not tested but french traductions were double-checked and correct

### CODE REVIEW

File: fermata/src/main/java/me/aap/fermata/ui/activity/MainActivityPrefs.java [hunk 1]
```diff
@@ -252,7 +252,8 @@ enum Lang {
 		HR,
 		JA,
 		ZH_TW(Locale.TRADITIONAL_CHINESE),
-		KO;
+		KO,
+		FR;
```
Comment (AndreyPavlenko) [inline fermata/src/main/java/me/aap/fermata/ui/activity/MainActivityPrefs.java:256]:
```suggestion
		FR(Locale.FRENCH);
```

**Comment by JUL1EN094** [comment]
@AndreyPavlenko I can also create french files in "modules" folder for chat/felex/tv/web with another PR. I just don't know if it is usefull (I'm not an android dev, don't know how it's working) . Let me know if it's usefull, it should not be too much effort for me

**Comment by AndreyPavlenko** [comment]
Sure, it's useful and should work in the same way as in the main module.

**Comment by JUL1EN094** [comment]
"modules" (chat,felex,tv,web) french translations added in this PR as well.

**Comment by AndreyPavlenko** [comment]
Great work! Thanks!

**Comment by AndreyPavlenko** [comment]
Please check - https://app.box.com/s/ns86n7vc3ssaltli0mmibrydnyvbr6bs .

**Pull request merged**